internal/plugin/manager.go

```diff
@@ -45,16 +45,16 @@ func (m *Manager) ApplyOptions(opts ...managerOption) {
 	}
 }
 
-func (m *Manager) PluginPackages(inputs []*nix.Input) ([]*nix.Input, error) {
-	pkgs := []*nix.Input{}
+func (m *Manager) PluginInputs(inputs []*nix.Input) ([]*nix.Input, error) {
```

Comment:
Same here

Reply:
> nomenclature: I have tried to ensure consistency around:
inputs referring to nix.Input
flakeInputs referring to shellgen.flakeInput
packages referring to raw strings that are in the packages list in devbox.json.